services/src/main/java/com/linkedin/kmf/services/configs/CommonServiceConfig.java

```diff
@@ -27,4 +27,13 @@
     + " created or rebalanced.  ceil(nBrokers * partitionsToBrokerRatio) is used to determine the actual number of "
     + "partitions when partitions are added or removed.";
 
+  public static final String TOPIC_REPLICATION_FACTOR_CONFIG = "topic-management.replicationFactor";
+  public static final String TOPIC_REPLICATION_FACTOR_DOC = "When a topic is created automatically this is the "
+      + "replication factor used.";
+
+  public static final String TOPIC_CREATION_ENABLED_CONFIG = "topic-management.topicCreationEnabled";
```

Comment:
Given that we have already release kafka-monitor as open source, let's no make this backward incompatible change by changing the config name, unless it is absolutely necessary for new use-case.

Besides, if it is a common config used by ProduceService and TopicManagementService, this config name should not have `topic-management` as prefix.